tensorpack/dataflow/dataset/pascalvoc12.py

```diff
@@ -0,0 +1,59 @@
+# Author: Tao Hu <taohu620@gmail.com>
+
+
+import os
+import gzip
+import numpy as np
+import cv2
+
+from ...utils import logger
+from ..base import RNGDataFlow
+
+__all__ = ['PascalVOC12']
+
+
+class PascalVOC12(RNGDataFlow):
```

Comment:
What files/directory structure does it require? What data (the meaning, shape, scale) does it produce? Could you add these documentation and possible send a separate PR for it?
It's hard to work with such a large PR. No matter whether the example works or not, a popular dataset loader alone can be a great feature to add.

Reply:
still work on documentation. if possible, I will send a seperate pr.